### ISSUE TITLE
all training data for API Retriever have label 1

### ISSUE THREAD

**Issue by ZongqianLi** [OpenBMB/ToolBench#241]
In page 8 of the paper, it says: "During training, we regard the relevant APIs of each instruction generated in 2.2 as positive examples and also sample a few APIs as negative examples for contrastive learning"
However, in toolbench/retrieval/train.py, train_samples only have one label value, 1.
Could I ask whether I have missed the contrastive learning or it is not in the codes?
